src/bicep/examples/automationAccount/automationAccount.bicep

```diff
@@ -0,0 +1,64 @@
+/*
+Deployes an automation account with modules/runbook/diagnostic logs
+Makes use of example azure automation bicep: https://github.com/Azure/bicep/tree/main/docs/examples/301/automation-account-import-runbooks-and-modules
+*/
+targetScope = 'subscription'
+
+param mlzDeploymentVariables object = json(loadTextContent('../deploymentVariables.json'))
+
+@description('The name of the automation account which will be created.  If unchanged or not specified, the MLZ resource prefix + "-aAc" will be utilized.')
+param automationAcctName string = '${mlzDeploymentVariables.mlzResourcePrefix.Value}-aAc'
+
+@description('The name of the resource group in which the automation account will be deployed. If unchanged or not specified, the MLZ operations resource group is used.')
+param targetResourceGroup string = '${mlzDeploymentVariables.spokes.Value[1].resourceGroupName}'
+
+@description('The URL location to the powershell runbook you wish to use')
+param automationRunbook string = 'https://raw.githubusercontent.com/Azure/missionlz/breestryker/AzAutomationBicepEx/src/bicep/examples/automationAccount/test.ps1'
```

Comment:
This URL points to the current branch, which will be gone when this PR is finished. 

Should this be an optional array parameter? That would allow users to provide array of runbooks to add, or not specify any runbooks and just deploy the automation account. The code sample in the README could provide the optional parameter and point to a public URI from the AZ Automation examples, but by default this template would not deploy a runbook. There could be other ways to handle this too.

Reply:
I left a note that we need to either remove the default or update the default.  I am okay with any of the above recommendations.  So whatever is preferred.